From 742ee4c497d4e0a1071a215bf22ba5944a3b8144 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 19 Apr 2025 12:03:42 +0000
Subject: [PATCH] application: Present main window on activation

This raises the main window to the front if already exists.
---
 src/application.vala | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/application.vala b/src/application.vala
index b2c02cc..5a5ca8b 100644
--- a/src/application.vala
+++ b/src/application.vala
@@ -106,6 +106,7 @@ namespace Connections {
             base.activate ();
 
             if (main_window != null) {
+                main_window.present ();
                 return;
             }
 
-- 
GitLab

